homeassistant/components/androidtv/media_player.py

```diff
@@ -224,6 +224,7 @@ def __init__(self, aftv, name, apps, turn_on_command,
             self.exceptions = (ConnectionResetError, RuntimeError)
 
         # Property attributes
+        self._adb_response = ''
```

Comment:
Is there some reason to use empty string instead of `None`?

Reply:
I could use `None` instead, if that would be preferable.

Reply:
I switched to using `None` instead of an empty string. So if there wasn't any output, the states page will report that the `adb_response` attribute is `null`.